### ISSUE TITLE
bug: Paths should be relative to the import paths

### ISSUE THREAD

**Issue by buehler** [riok/Kreya#16]
**Describe the bug**
I created a file importer on windows 10 which had a relative filepath:
```
 "importStreams": [
    {
      "id": "36ca3c45-96b2-4f9c-97ae-234b51c42363",
      "name": "Grpc Files",
      "importerName": "ProtoFile",
      "options": {
        "files": [
          "..\\src\\Grpc"
        ],
        "importPaths": []
      }
    }
```

When opening this kreya project on a mac, the error below happens.

**To Reproduce**
1. Create an importer for grpc files on windows 10
2. open the kreya project on a mac

**Additional context**
```
{
  "kreyaVersion": "1.1.0",
  "platform": "MacIntel",
  "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko)"
}
System.ArgumentException: Paths should be relative to the import paths, not rooted (Parameter 'name')
   at Google.Protobuf.Reflection.FileDescriptorSet.Add(String name, Boolean includeInOutput, TextReader source, FileDescriptorProto fromFile)
   at Kreya.Grpc.Core.Importer.ProtoFile.ProtoFileImporter.AddFiles(IEnumerable`1 files, FileDescriptorSet set)
   at Kreya.Grpc.Core.Importer.ProtoFile.ProtoFileImporter.Import(IImportStream importStream, ProtoFileImporterOptions options)
   at Kreya.Core.Importer.ImporterService.RunSingleImporter(ImportStream importStream)
   at Kreya.Core.Importer.ImporterService.RunImporters()
   at SpiderEye.Bridge.ApiMethod.InvokeAsync(Object instance, Object parameter)
   at SpiderEye.Bridge.WebviewBridge.InvokeWithDependencyInjection(IDependencyInjectionApiMethod apiMethod, Object parameters)
   at SpiderEye.Bridge.WebviewBridge.InvokeWithDependencyInjection(IDependencyInjectionApiMethod apiMethod, Object parameters)
   at SpiderEye.Bridge.WebviewBridge.ResolveCall(String id, String parameters)
```

**Comment by CommonGuy** [comment]
We currently store paths with the OS specific directory separator. Creating files on Windows (where the separator is `\`) and opening them on Mac/Linux (where the separator is `/`) creates the bug.

Going forward, we'll store relative paths with a forward slash, since that is also supported by Windows. We will also normalize paths created by older versions to fix this bug.

**Comment by CommonGuy** [comment]
This should be fixed with V1.2.0, which we just released.
